# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/zzz/hotfix-inst.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- zzz-0.2.0/Makefile.vanilla	2026-09-08 22:10:23.233656472 +0300
+++ zzz-0.2.0/Makefile	2026-09-08 22:12:20.705438339 +0300
@@ -71,7 +71,8 @@
 
 #: Install man pages into $DESTDIR/$mandir/man*/.
 install-man: build-man
-	$(INSTALL) -D -m644 -t $(DESTDIR)$(mandir)/man8/ $(addprefix $(D)/,$(filter %.8,$(MAN_FILES)))
+	$(INSTALL) -D -m644 build/ZZZ.8 $(DESTDIR)$(mandir)/man8/
+	$(INSTALL) -D -m644 build/zzz.8 $(DESTDIR)$(mandir)/man8/
 
 #: Uninstall from $DESTDIR.
 uninstall:
